site/next.config.js

```diff
@@ -4,8 +4,8 @@ const withFonts = require("next-fonts");
 module.exports = withFonts(
   withImages({
     pageExtensions: ["page.ts", "page.tsx", "api.ts"],
-    webpack5: false,
     optimizeFonts: true,
+    swcMinify: true,
```

Comment:
Context:
- https://nextjs.org/blog/next-12#faster-builds-and-fast-refresh-with-rust-compiler
- https://nextjs.org/docs/messages/swc-minify-enabled

We can disable this option later if it does now work well for us.

Reply:
UPD this option did not work well. I was getting this on `/docs` and `/spec` pages:

```
TypeError: Cannot read properties of undefined (reading 'type')
```

<img width="581" alt="Screenshot 2022-02-14 at 13 39 39" src="https://user-images.githubusercontent.com/608862/153874816-8513bff8-24a7-44a6-b9f1-e9e30b2427c0.png">

 Demo: https://blockprotocol-6iteluctz-hashintel.vercel.app/docs

 Issue to watch: https://github.com/vercel/next.js/issues/30379

Reply:
How can I disable retry logic for `GEThttps://blockprotocol-6iteluctz-hashintel.vercel.app/api/me`?

I need to read the content of the function `j`, but browsers are not showing because of the infinite loop.

Reply:
Nevermind, I somehow got the content using firefox developer edition :)

Reply:
Hey @kdy1, thanks for your attention to our repo! Glad to see the results of your investigation in https://github.com/swc-project/swc/pull/3585!